internal/db/schema/migrations/oss/postgres/99/02_worker_controller_tables.up.sql

```diff
@@ -68,6 +50,28 @@ create table server_worker_tag (
 -- Aaand drop server_tag
 drop table server_tag;
 
+create table server_controller (
```

Comment:
is there a reason `server_controller` was moved below `server_worker`? it makes this diff harder to review